### ISSUE TITLE
Accessibility: Complete WCAG compliance in all Elementor widgets and theme builder

### ISSUE THREAD

**Issue by tomas-eklund** [elementor/elementor#11779]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
In more and more countries, including [the USA](https://en.wikipedia.org/wiki/Americans_with_Disabilities_Act_of_1990) and the countries of the [European Union](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2016.327.01.0001.01.ENG), it is a legal requirement to conform to the [WCAG standard](https://www.w3.org/TR/WCAG21/) for accessibility. This applies to many web sites, although not all. It's reasonable to expect that more and more web sites will be bound by law to conform as the years goes by. Non-conformance is, in practice, discrimination and can lead to hefty fines and bad publicity. 

Unfortunately, Elementor is not fully compliant out of the box and breaks many of the rules for accessibility. 

Building a WCAG compliant web site using Elementor today is possibly more work than it's worth since the tool is working _against_ you and not _for_ you. 

**Describe the solution you'd like**
A site built with Elementor Pro and Hello Elementor theme, using the default settings of all widgets, should be WCAG compliant out of the box. At least to the extent possible. 

I suggest assigning a group within the Elementor company to review the topic of accessibility in Elementor and what can be done to make Elementor a tool of choice for agencies tasked with building web sites that are required by law to conform to the WCAG accessibility guidelines. 

**Describe alternatives you've considered**
Building web sites without Elementor.

**Comment by shimsa** [comment]
+1

**Comment by KobiCWW** [comment]
+1 super important

**Comment by sisland** [comment]
+1 A must have in 2020

**Comment by Sumner10** [comment]
+1 - Essential!

**Comment by d0ky** [comment]
+1

**Comment by eugenem** [comment]
+100500

even drop-down menu is not accessible: you can navigate with tabs into it while it's closed...

**Comment by rj-csi** [comment]
+1
Working on getting our site WCAG 2.0 compliant by Jan 2021 as it's a legal requirement - running into issues with some widgets such as the search box which doesn't have a label.

**Comment by ByteOutfitters** [comment]
+1 this is critical

**Comment by besweeet** [comment]
It would probably be advantageous for Elementor to become more accessible so that it can be something they brag about and get more sales from.

**Comment by DorShahar** [comment]
This issue has been resolved in Elementor v3.1.0
Feel free to update
Thanks!

**Comment by tomas-eklund** [comment]
I feel that it is very unfortunate that this ticket has been closed before any significant effort has been made to adress the situation. The issue of accesibility is a burning question and a very important consideration for web agencies world wide. Elementor is currently a bad choice for clients affected by these legal requirements.

**Comment by amesplant-dmv** [comment]
I agree that this issue should be far from closed. Currently running a site built with Elementor and it fails in Accessibility in so many ways. At this point, we will likely be recommending Gutenberg, because the fix for the numerous accessibility issues is insurmountable.

**Comment by MGACreative** [comment]
I agree. Some are easy fixes such as turning off default settings like animations or open in a new tab. Others like buttons and the aria role need to be fixed.

**Comment by besweeet** [comment]
Why was this closed? Elementor still very much lacks proper accessibility. The latest example is how the video widget cannot be used with keyboard navigation.

**Comment by sisland** [comment]
This is closed because is not a priority for them. Making WCAG compliance is a lot of work for them so seems they are no longer interested...

**Comment by mbaierl** [comment]
With the new directive coming up in the EU this is going to the a huge issue!
I can't do a project right now with Elementor because it will fail several WCAG tests...
 @DorShahar
